src/Agent.Worker/Release/Artifacts/CustomArtifact.cs

```diff
@@ -169,7 +169,9 @@ private static string GetArtifactStreamType(CustomArtifactDownloadDetails artifa
 
         private static HttpWebResponse GetWebResponse(IExecutionContext executionContext, string url, IEndpointAuthorizer authorizer)
         {
+            #pragma warning disable SYSLIB0014 // TODO: create GitHub issue to track this warning
```

Comment:
We shouldn't add a "TODO"'s in code; they tend to live forever

In terms of creating a github issue; I'm thinking it would just be noise